daprdocs/content/en/reference/components-reference/supported-configuration-stores/_index.md

```diff
@@ -5,7 +5,7 @@ linkTitle: "Configuration stores"
 weight: 4500
 description: The supported configuration stores that interface with Dapr
 aliases:
-  - "/operations/components/setup-secret-store/supported-configuration-stores/"
+  - "/operations/components/setup-configuration-store/supported-configuration-stores/"
```

Comment:
@greenie-msft Is this alias being used anywhere and if so it will break ... IMO this change is correct but will it break ?

Reply:
I did not find any other refs in the repo.

Reply:
not in repo but external refs ... again a call to be taken by @dapr/maintainers-docs

Reply:
Most likely not used externally. I would change it